ast/format.go

```diff
@@ -0,0 +1,356 @@
+package ast
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// Returns a valid query for a given AST rooted at node `n`.
+func Format(n Node) string {
+	f := &formatter{new(strings.Builder)}
+	f.formatNode(n)
+	return f.get()
+}
+
+type formatter struct {
+	*strings.Builder
+}
+
+func (f *formatter) get() string {
+	return f.String()
+}
+
+// strings.Builder's methods never returns a non-nil error.
+func (f *formatter) writeString(s string) {
+	_, err := f.WriteString(s)
+	if err != nil {
+		panic(err)
+	}
+}
```

Comment:
WriteString is documented to always return a `nil` error. https://golang.org/pkg/strings/#Builder.WriteString Can you update this method to ignore both return values and make a comment about why it is safe?